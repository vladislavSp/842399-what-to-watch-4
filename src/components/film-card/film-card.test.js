import React from 'react';
import renderer from 'react-test-renderer';
import FilmCard from './film-card';

const title = `Fantastic Beasts`;

it(`renders correctly`, () => {
  const tree = renderer.create(<FilmCard
    title={title}
    handleTitleFocus={() => {}}>
  </FilmCard>)
  .toJSON();
  expect(tree).toMatchSnapshot();
});
