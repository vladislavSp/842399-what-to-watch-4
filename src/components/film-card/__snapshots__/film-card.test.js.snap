// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<article
  className="small-movie-card catalog__movies-card"
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
>
  <a
    href="/films/0"
    onClick={[Function]}
  >
    <div
      className="small-movie-card__image"
    >
      <video
        autoPlay={false}
        height="175"
        poster="https://htmlacademy-react-3.appspot.com/wtw/static/film/preview/gangs_of_new_york.jpg"
        width="280"
      />
    </div>
  </a>
  <h3
    className="small-movie-card__title"
  >
    <a
      className="small-movie-card__link"
      href="/films/0"
      onClick={[Function]}
    >
      Gangs of new york
    </a>
  </h3>
</article>
`;
